### PR TITLE
toNativeObject does not return null values

### PR DESCRIPTION
**Pull request by anatol-sialitski**

### CODE REVIEW

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+ * @param {string} params.query Query expression.
+ * @param {object} [params.filters] Filters to apply to query result
+ * @param {string} [params.sort] Sorting expression.
+ * @param {string} [params.aggregations] Aggregations expression.
+ * @param {string[]} [params.contentTypes] Content types to filter on.
+ *
+ * @returns {Object} Result of query.
+ */
+exports.query = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.QueryContentHandler');
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:169]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 19
           

Complexity increasing per file
==============================
- modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js  2
- modules/script/script-impl/src/main/java/com/enonic/xp/script/impl/util/ScriptMapGenerator.java  1
         

Clones added
============
- modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js  13
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265&bid=20402577)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+ * @example-ref examples/content/getAttachmentStream.js
+ *
+ * @param {object} params JSON with the parameters.
+ * @param {string} params.key Path or id to the content.
+ * @param {string} params.name Attachment name.
+ *
+ * @returns {*} Stream of the attachment data.
+ */
+exports.getAttachmentStream = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetAttachmentStreamHandler');
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:90]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+ * Check if content exists.
+ *
+ * @example-ref examples/content/exists.js
+ *
+ * @param {string} [params.key] content id.
+ *
+ * @returns {boolean} True if exist, false otherwise.
+ */
+exports.exists = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.ContentExistsHandler');
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:191]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+/**
+ * Returns the properties and icon of the specified content type.
+ *
+ * @example-ref examples/content/getType.js
+ *
+ * @param name Name of the content type, as 'app:name' (e.g. 'com.enonic.myapp:article').
+ * @returns {ContentType} The content type object if found, or null otherwise. See ContentType type definition below.
+ */
+exports.getType = function (name) {
+    var bean = __.newBean('com.enonic.xp.lib.content.ContentTypeHandler');
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:225]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+ *
+ * @example-ref examples/content/getSite.js
+ *
+ * @param {object} params JSON with the parameters.
+ * @param {string} params.key Path or id to the content.
+ *
+ * @returns {object} The current site as JSON.
+ */
+exports.getSite = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetSiteHandler');
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:107]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 6]
```diff
@@ -0,0 +1,255 @@
...
+exports.query = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.QueryContentHandler');
+    bean.start = params.start;
+    bean.count = params.count;
+    bean.query = nullOrValue(params.query);
+    bean.sort = nullOrValue(params.sort);
+    bean.aggregations = __.toScriptValue(params.aggregations);
+    bean.contentTypes = __.toScriptValue(params.contentTypes);
+    bean.filters = __.toScriptValue(params.filters);
+    bean.highlight = __.toScriptValue(params.highlight);
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:177]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 7]
```diff
@@ -0,0 +1,255 @@
...
+ * Functions to find and manipulate content.
+ *
+ * @example
+ * var contentLib = require('/lib/xp/content-graphql');
+ *
+ * @module content
+ */
+
+function required(params, name) {
+    var value = params[name];
```
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:11]: Codacy found an issue: [Variable Assigned to Object Injection Sink](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

**Comment by rymsha** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 19
           

Complexity increasing per file
==============================
- modules/portal/portal-impl/src/test/java/com/enonic/xp/portal/impl/controller/PortalResponseSerializerTest.java  1
- modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js  2
- modules/script/script-impl/src/main/java/com/enonic/xp/script/impl/util/ScriptMapGenerator.java  1
         

Clones added
============
- modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js  13
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264&bid=20402576)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 8]
```diff
@@ -0,0 +1,255 @@
...
+/**
+ * Returns the list of all the content types currently registered in the system.
+ *
+ * @example-ref examples/content/getTypes.js
+ *
+ * @returns {ContentType[]} Array with all the content types found. See ContentType type definition below.
+ */
+exports.getTypes = function () {
+    var bean = __.newBean('com.enonic.xp.lib.content.ContentTypeHandler');
+    return __.toNativeObjectIncludeNullValues(bean.getAllContentTypes());
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:239]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 5 above)
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:107]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 9]
```diff
@@ -0,0 +1,255 @@
...
+ * This function returns a content attachments.
+ *
+ * @example-ref examples/content/getAttachments.js
+ *
+ * @param {string} key Path or id to the content.
+ *
+ * @returns {object} An object with all the attachments that belong to the content, where the key is the attachment name. Or null if the content cannot be found.
+ */
+exports.getAttachments = function (key) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetAttachmentsHandler');
```
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:73]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 10]
```diff
@@ -0,0 +1,255 @@
...
+ *
+ * @returns {Object} Result of query.
+ */
+exports.query = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.QueryContentHandler');
+    bean.start = params.start;
+    bean.count = params.count;
+    bean.query = nullOrValue(params.query);
+    bean.sort = nullOrValue(params.sort);
+    bean.aggregations = __.toScriptValue(params.aggregations);
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:174]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 11]
```diff
@@ -0,0 +1,255 @@
...
+
+/**
+ * Returns outbound dependencies on a content.
+ *
+ * @param {object} params JSON parameters.
+ * @param {string} params.key Path or id of the content.
+ * @returns {object} Content Ids.
+ */
+exports.getOutboundDependencies = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetOutboundDependenciesHandler');
```
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:250]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 12]
```diff
@@ -0,0 +1,255 @@
...
+ * @param {object} params JSON with the parameters.
+ * @param {string} params.key Path or id to the parent content.
+ * @param {number} [params.start=0] Start index (used for paging).
+ * @param {number} [params.count=10] Number of contents to fetch.
+ * @param {string} [params.sort] Sorting expression.
+ *
+ * @returns {Object} Result (of content) fetched from the repository.
+ */
+exports.getChildren = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetChildContentHandler');
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:144]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 8 above)
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:239]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 13]
```diff
@@ -0,0 +1,255 @@
...
+ * @returns {Object} Result of query.
+ */
+exports.query = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.QueryContentHandler');
+    bean.start = params.start;
+    bean.count = params.count;
+    bean.query = nullOrValue(params.query);
+    bean.sort = nullOrValue(params.sort);
+    bean.aggregations = __.toScriptValue(params.aggregations);
+    bean.contentTypes = __.toScriptValue(params.contentTypes);
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:175]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 3 above)
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:191]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 14]
```diff
@@ -0,0 +1,255 @@
...
+ *
+ * @example-ref examples/content/getType.js
+ *
+ * @param name Name of the content type, as 'app:name' (e.g. 'com.enonic.myapp:article').
+ * @returns {ContentType} The content type object if found, or null otherwise. See ContentType type definition below.
+ */
+exports.getType = function (name) {
+    var bean = __.newBean('com.enonic.xp.lib.content.ContentTypeHandler');
+    bean.name = nullOrValue(name);
+    return __.toNativeObjectIncludeNullValues(bean.getContentType());
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:227]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:227]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 7 above)
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:11]: Codacy found an issue: [Variable Assigned to Object Injection Sink](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 6 above)
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:177]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 15]
```diff
@@ -0,0 +1,255 @@
...
+ * @example-ref examples/content/getSiteConfig.js
+ *
+ * @param {object} params JSON with the parameters.
+ * @param {string} params.key Path or id to the content.
+ * @param {string} params.applicationKey Application key.
+ *
+ * @returns {object} The site configuration for current application as JSON.
+ */
+exports.getSiteConfig = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetSiteConfigHandler');
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:124]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 16]
```diff
@@ -0,0 +1,255 @@
...
+ */
+exports.query = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.QueryContentHandler');
+    bean.start = params.start;
+    bean.count = params.count;
+    bean.query = nullOrValue(params.query);
+    bean.sort = nullOrValue(params.sort);
+    bean.aggregations = __.toScriptValue(params.aggregations);
+    bean.contentTypes = __.toScriptValue(params.contentTypes);
+    bean.filters = __.toScriptValue(params.filters);
```
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:176]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 17]
```diff
@@ -0,0 +1,255 @@
...
+ * Gets permissions on a content.
+ *
+ * @example-ref examples/content/getPermissions.js
+ *
+ * @param {object} params JSON parameters.
+ * @param {string} params.key Path or id of the content.
+ * @returns {object} Content permissions.
+ */
+exports.getPermissions = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetPermissionsHandler');
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:208]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 15 above)
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:124]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 18]
```diff
@@ -0,0 +1,255 @@
...
+ * @example-ref examples/content/get.js
+ *
+ * @param {object} params JSON with the parameters.
+ * @param {string} params.key Path or id to the content.
+ * @param {string} [params.versionId] Version Id of the content.
+ *
+ * @returns {object} The content (as JSON) fetched from the repository.
+ */
+exports.get = function (params) {
+    var bean = __.newBean('com.enonic.xp.lib.content.GetContentHandler');
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:57]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 2 above)
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:90]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 9 above)
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:73]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 4 above)
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:225]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 16 above)
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:176]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 1 above)
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:169]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 12 above)
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:144]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 11 above)
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:250]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 18 above)
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:57]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 19]
```diff
@@ -0,0 +1,255 @@
...
+    bean.key = required(params, 'key');
+    bean.versionId = nullOrValue(params.versionId);
+    return __.toNativeObjectIncludeNullValues(bean.execute());
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:60]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:60]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 13 above)
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:175]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 17 above)
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:208]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 10 above)
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:174]: Codacy found an issue: ['__' is not defined.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js  2
- modules/script/script-impl/src/main/java/com/enonic/xp/script/impl/util/ScriptMapGenerator.java  1
         

Clones added
============
- modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js  13
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265&bid=20402577)

**Comment by rymsha** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- modules/script/script-impl/src/main/java/com/enonic/xp/script/impl/util/ScriptMapGenerator.java  1
- modules/portal/portal-impl/src/test/java/com/enonic/xp/portal/impl/controller/PortalResponseSerializerTest.java  1
- modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js  2
         

Clones added
============
- modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js  13
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264&bid=20402576)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js [hunk 20]
```diff
@@ -0,0 +1,257 @@
...
+ * @example
+ * var contentLib = require('/lib/xp/content-graphql');
+ *
+ * @module content
+ */
+
+/* global __ */
+
+function required(params, name) {
+    var value = params[name];
```
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:13]: Codacy found an issue: [Variable Assigned to Object Injection Sink](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:13]: Codacy found an issue: [Variable Assigned to Object Injection Sink](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

**Comment by jsi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js  2
- modules/script/script-impl/src/main/java/com/enonic/xp/script/impl/util/ScriptMapGenerator.java  1
         

Clones added
============
- modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js  13
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265&bid=20402577)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 20 above)
Comment (jsi) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:13]: Codacy found an issue: [Variable Assigned to Object Injection Sink](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=6266265)

**Comment by rymsha** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- modules/script/script-impl/src/main/java/com/enonic/xp/script/impl/util/ScriptMapGenerator.java  1
- modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js  2
         

Clones added
============
- modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js  13
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264&bid=20402576)

File: modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js (hunk 20 above)
Comment (rymsha) [inline modules/lib/lib-content/src/main/resources/lib/xp/content-graphql.js:13]: Codacy found an issue: [Variable Assigned to Object Injection Sink](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=6266264)

**Review by sigdestad — CHANGES_REQUESTED** [review]
As discussed, I do not want to have presentation-specific things like this in the Core. If this should be fixed some where, it is in the framework (toNative..) or similar.

@rymsha any comments?

**Comment by rymsha** [comment]
This PR won't be accepted in how it's made now. We need to discuss further

**Comment by rymsha** [comment]
The main question here, why do we swallow null values in initial implementation?

**Comment by alansemenov** [comment]
This issue was caused by [this](https://github.com/enonic/lib-guillotine/issues/65) and was fixed [here](https://github.com/enonic/starter-headless/pull/9), so this PR can be closed. We shouldn't swallow the null values but fixing this can cause side effects, so I suggest leaving it for now.

**Pull request closed without merging**